server/service.go

```diff
@@ -369,6 +372,11 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 				relays[blockHash] = append(relays[blockHash], relay.String())
 			}
 
+			// Skip if value (fee) is not greater than minimum bid
```

Comment:
would "minimum bid amount" imply that the value would already be accepted, rather than only values greater than it?

Reply:
You are right that does make more sense.  Updated to accept bids equal to the minimum.